Changelog.md

```diff
@@ -1,6 +1,7 @@
 # Next
 ### Added
 - **Breaking Change** Added a `.requestCustomJSONEncodable` case to `Task`. [#1443](https://github.com/Moya/Moya/pull/1443) by [@evgeny-sureev](https://github.com/evgeny-sureev).
+- **Breaking Change** Added `failsOnEmptyData` boolean support for the `Decodable` map functions. [#1508](https://github.com/Moya/Moya/pull/1508) by [@jeroenbb94](https://github.com/Jeroenbb94).
```

Comment:
I know this isn't the main point of this PR, but is it really a breaking change? There is a default argument for this parameter, this parameter is a last one, and if you don't provide an option you will get the same behavior as in previous versions. Or am I missing something? 🤔

Reply:
Sorry @sunshinejr that was me 😅
https://github.com/Moya/Moya/pull/1508#discussion_r157958053

Reply:
My definition of breaking is anything that will prevent the project from compiling. Variable name changes, type name changes, function signature changes, adding cases to an enum, or changes to a more restrictive form of access control.

Reply:
Oh! You are absolutely correct @SD10, I just don't see any issue that could break it, do you have anything particular in mind?

Reply:
Nothing in particular, I just go by those general rules. If someone is passing `Response.map` as an argument to a higher order function, the receiving method will need to change the type of its parameter.